### DISCUSSION TITLE
ASIO / Routing Support

### DISCUSSION THREAD

**Discussion by ubermunchen** [apohl79/audiogridder#278] [Ideas]
In the long term, it would be kinda cool if asio or routing could be supported for the audio transfer between the host and slave as an alternative to the current protocol. It would allow people to elect to use other protocols, e.g. dante, avb, etc. By way of example, Presonus has a plugin that allows external routing and return called pipeline xt. It uses a ping to auto adjust for round trip latency. As long as you allow the user to select to use their own asio devices, I don't see any licensing issues. It's no different than what every daw does. Maybe you could even wrap the asio being used by your plugin so it can be different than the asio being used by the daw, e.g. people could license dante virtual sound card to use as your back end while using a firewire or USB device for monitoring. This would also allow your software to piggyback on existing audio network protocols of the users choice, including the current protocol.

**Comment by apohl79** [comment]
I like the idea!

**Comment by michalsykora** [comment]
how is this going? :)
in my case it would be some reverbs on Mac Mini, Reaper, AG, DVS
